tools/build_node.js

```diff
@@ -6,7 +6,35 @@ const { filter } = require("./lib/dependencies");
 const { rollupWrite } = require("./lib/bundling.js");
 const log = (...args) => console.log(...args);
 
-async function buildNodeIndex(languages) {
+async function buildESMIndex(name, languages) {
+  const header = `import hljs from './core.mjs';`;
+  const footer = "export default hljs;";
+
+  const safeImportName = (s) => {
+    s = s.replace(/-/g, "_");
+    if (/^\d/.test(s)) s = `L_${s}`;
+    return s;
+  };
+
+  const registration = languages.map((lang) => {
+    let out = '';
+    const importName = safeImportName(lang.name);
+    let require = `import ${importName} from './languages/${lang.name}.js';`;
+    // TODO: break this with v11? All modules must export default?
+    if (lang.loader) {
+      require = require += `.${lang.loader}`;
```

Comment:
I'm not sure to understand what this is doing? If you are not updating `out`, it is simply a no-op right?
```suggestion
      require += `.${lang.loader}`;
```

Reply:
Ugh yeah, this didn't come across right - but I'm just going to remove those bits entirely anyways.